SECP-service/src/main/resources/assets/app/views/login/login.html

```diff
@@ -0,0 +1,26 @@
+<div class="container-fluid">
+
+    <div class="form-group">
+        <form autocomplete="on">
+            <h1 id="formHeader">Log in</h1>
+
+            <div class="group">
+                <input type="email" required="required" onkeyup="this.setAttribute('value', this.value);" value="" />
+                <span class="highlight"></span>
+                <span class="bar"></span>
+                <label>Email</label>
```

Comment:
I think you should change this to username. The server side accepts username not email.

Reply:
You're going to have to change the high level docs to reflect this. Basically all the use cases that involve emails need to change to username.